public/app/plugins/datasource/jaeger/package.json

```diff
@@ -0,0 +1,49 @@
+{
```

Comment:
Honestly not sure if we need all these dependencies, probably some can be removed?

Reply:
Removed a few of them.